src/download_tools/utils.ts

```diff
@@ -113,6 +113,23 @@ export function findAriaFilePath(files: any[]): string {
  * @param {any[]} files The list of files in the download
  * @returns {StatusMessage} An object containing a printable status message and the file name
  */
+
+function timeLeft (totalLength: number, speed: number) {
+ var time = (totalLength/1048576)/((speed/125000)/8);
```

Comment:
Why use different factors to scale the speed and length? Why scale them at all? Length should be in B, and speeds should be in B/s, so there will be no scaling required.

Reply:
cz it works xD

Reply:
czwhynot xD
also heroku was giving some errors so needed that otherwise it not works

Reply:
verified it works xD

Reply:
Even `sudo rm -rf / --no-preserve-root` works. Doesn't mean it should be used.

Reply:
> Even `sudo rm -rf / --no-preserve-root` works. Doesn't mean it should be used.

This.